packages/js/client/src/__tests__/e2e/test-cases.ts

```diff
@@ -412,7 +412,7 @@ export const runInvalidTypesTest = async (
   );
 };
 
-export const runJsonTypeTest = async (client: PolywrapClient, uri: string) => {
+export const runJsonTypeTest = async (client: PolywrapClient, uri: string, testReserved: boolean = false) => {
```

Comment:
Can you please pass in `true` for `testReserved` in the wasm-rs tests? This logic is not currently being used.